packages/dapp-svelte-wallet/api/src/types.js

```diff
@@ -86,8 +86,8 @@
  * @property {Petname} pursePetname the petname for this purse
  * @property {any} displayInfo the brand's displayInfo
  * @property {any} value the purse's current balance
- * @property {any} currentAmountSlots
- * @property {any} currentAmount
+ * @property {any} recentAmountSlots TODO do we also need a deprecated currentAmountSlots?
+ * @property {any} recentAmount TODO do we also need a deprecated currentAmount?
```

Comment:
Note: I need advice here as explained in the TODO comment.

Reply:
I didn't see them used anywhere except in the Wallet UI, so I suspect the answer is "no", but I will leave it up to @katelynsills when she takes this on.

Reply:
IMO, we don't need to keep a deprecated currentAmountSlots